src/content/docs/en/reference/api-reference.mdx

```diff
@@ -880,10 +880,12 @@ import { Code } from 'astro/components';
 ---
 <!-- Syntax highlight some JavaScript code. -->
 <Code code={`const foo = 'bar';`} lang="js" />
-<!-- Optional: customize your theme. -->
+<!-- Optional: Customize your theme. -->
 <Code code={`const foo = 'bar';`} lang="js" theme="dark-plus" />
 <!-- Optional: Enable word wrapping. -->
 <Code code={`const foo = 'bar';`} lang="js" wrap />
+<!-- Optional: Output inline code. -->
+<Code code={`const foo = 'bar';`} lang="js" inline />
```

Comment:
```suggestion
Here is <Code code={`const foo = 'bar';`} lang="js" inline /> available inline.
```
I'm not sold on the wording, but I think demonstrating that the component is meant to be used inline, as per the [original, original illustration of the toggle](https://github.com/withastro/astro/issues/5595#issuecomment-1397731722) is more helpful. If you can think of a better sentence, please do!  :sweat_smile:

Reply:
I kinda miss the alignment though with this 😄 maybe if we want to show it's inline, how about:
```suggestion
<span>
  Here is an inline example:
  <Code code={`const foo = 'bar';`} lang="js" inline />
</span>
```

Reply:
Wouldn’t a block-level wrapper element like `<p>` make more sense than `<span>` here?

```suggestion
<p>
  <Code code={`const foo = 'bar';`} lang="js" inline />
  will be rendered inline.
</p>
```

Reply:
Ah you're right. I thought the issue was nesting block elements inside inline elements, which `span` easily identifies as an inline element, but the issue seems to be specific to `<p>` with it's spec strictly preventing block elements while `<span>` is more permissive.